docs/docs/guide/09-interchange/02-init.md

```diff
@@ -126,50 +126,86 @@ Create the `mars.yml` file with your content:
 
 ```yaml
 # mars.yml
+version: 1
+build:
+  proto:
+    path: proto
+    third_party_paths:
+    - third_party/proto
+    - proto_vendor
 accounts:
-  - name: alice
-    coins: [ "1000token", "100000000stake", "1000marscoin" ]
-  - name: bob
-    coins: [ "500token", "1000marscoin", "100000000stake" ]
-validator:
-  name: alice
-  staked: "100000000stake"
+- name: alice
+  coins:
+  - 1000token
+  - 100000000stake
+  - 1000marscoin
+- name: bob
+  coins:
+  - 500token
+  - 1000marscoin
+  - 100000000stake
 faucet:
   name: bob
-  coins: [ "5token", "100000stake" ]
+  coins:
+  - 5token
+  - 100000stake
+  host: 0.0.0.0:4500
 genesis:
-  chain_id: "mars"
-init:
-  home: "$HOME/.mars"
+  chain_id: mars
+validators:
+- name: alice
+  bonded: 100000000stake
+  home: $HOME/.mars
 ```
 
 Create the `venus.yml` file with your content:
 
 ```yaml
 # venus.yml
+version: 1
+build:
+  proto:
+    path: proto
+    third_party_paths:
+    - third_party/proto
+    - proto_vendor
 accounts:
-  - name: alice
-    coins: [ "1000token", "1000000000stake", "1000venuscoin" ]
-  - name: bob
-    coins: [ "500token", "1000venuscoin", "100000000stake" ]
-validator:
-  name: alice
-  staked: "100000000stake"
+- name: alice
+  coins:
+  - 1000token
+  - 1000000000stake
+  - 1000venuscoin
+- name: bob
+  coins:
+  - 500token
+  - 1000venuscoin
+  - 100000000stake
 faucet:
-  host: ":4501"
   name: bob
-  coins: [ "5token", "100000stake" ]
-host:
-  rpc: ":26659"
-  p2p: ":26658"
-  prof: ":6061"
-  grpc: ":9092"
-  grpc-web: ":9093"
-  api: ":1318"
+  coins:
+  - 5token
+  - 100000stake
+  host: :4501
 genesis:
-  chain_id: "venus"
-init:
-  home: "$HOME/.venus"
+  chain_id: venus
+validators:
+- name: alice
+  bonded: 100000000stake
+  app:
+    api:
+      address: :1318
+    grpc:
+      address: :9092
+    grpc-web:
+      address: :9093
+  config:
+    p2p:
+      laddr: :26658
+    rpc:
+      laddr: :26659
+      pprof_laddr: :6061
+  home: $HOME/.venus
+
 ```
 
 In the `venus.yml` file, you can see the specific `host` parameter that you can use to change the ports for various
```

Comment:
It's no longer `host` in version 1.

Reply:
Isn't it the other way around?

https://github.com/ignite/cli/blob/bb12a25b497f0ea67541a98774fd3d6d19a122d3/ignite/config/chain/config.go#L75-L82

Reply:
As far as I understand, both `port` and `host` are working as of `v1`, but we want to deprecate `port` in favor of `host`.

Reply:
I didn't know there was a backward compatibility, since we migrate the file automatically, I though this wasn't necessary.

But even with that, the thing is the docs lists the config files content and just after refers a field that is not present in those configs. That's confusing no? In version 0, the field was there.

Reply:
Sorry, I confused what you were talking about with `faucet.host`. I actually didn't rewrite the Init section yet, just made sure `config.yml`s are up-to-date, but here's an updated note: https://github.com/ignite/cli/pull/3283/commits/3665022dec99426fa0bacc7717ece08712f95cb4

Reply:
Perfect!